src/cljs/proton/layers/lang/javascript/README.md

```diff
@@ -0,0 +1,74 @@
+## JavaScript configuration layer
+
+Adds javascript support to proton.
+
+Includes following atom packages:
+
+- [atom-ternjs](https://atom.io/packages/atom-ternjs)
+- [javascript-snippets](https://atom.io/packages/javascript-snippets)
+- [language-javascript](https://atom.io/packages/language-javascript)
+- [autocomplete-modules](https://atom.io/packages/autocomplete-modules)
+- [docblockr](https://atom.io/packages/docblockr)
+- [react](https://atom.io/packages/react)
+- [react-snippets](https://atom.io/packages/react-snippets)
+
+### Install
+
+Add `:lang/javascript` to your layers.
+For linters support add `:tools/linters`.
+
+### Layer Configuration
+
+Name                            | Default  | Type       | Description
+--------------------------------|----------|------------|-------------------------------
+`proton.lang.javascript.linter` | "eslint" | __string__ | Which linter to use by default
```

Comment:
I think this should be keyword instead of string to stay consistent with other layers